<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Интерфейс эксперта</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="uiTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="uiVariablesTab">
     <attribute name="title">
      <string>Переменные</string>
     </attribute>
     <widget class="QPushButton" name="uiCreateVariableButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QListWidget" name="uiTermsList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="uiAddTermButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QComboBox" name="uiHedgesCombo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiVariableLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Лингвистическая переменная:</string>
      </property>
     </widget>
     <widget class="QLabel" name="uiHedgesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Доступные модификаторы:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="uiTermsCombo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="uiRemoveTermButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Убрать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uiAddHedgeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uiRemoveHedgeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Убрать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uiDeleteVariableButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uiRenameVariableButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>65</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Переименовать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="uiVariablesCombo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="uiTermsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Доступные атомарные термы:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="uiHedgesList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiRangeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Диапазон значений базовой переменной:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiRangeMinEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiRangeMaxEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiRangeSepLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uiCommitVariableButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить изменения переменной</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="uiTermsTab">
     <attribute name="title">
      <string>Термы</string>
     </attribute>
     <widget class="QPushButton" name="delete_variable_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QComboBox" name="variables_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Атомарный терм:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_variable_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rename_variable_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>65</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Переименовать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Функция принадлежности:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="variables_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Характерные точки:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="uiHedgesTab">
     <attribute name="title">
      <string>Модификаторы</string>
     </attribute>
     <widget class="QPushButton" name="create_variable_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rename_variable_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>65</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Переименовать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="variables_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>391</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_variable_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>65</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Значение:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Модификатор:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
